repl/src/main/scala/com/cloudera/livy/repl/SparkRInterpreter.scala

```diff
@@ -126,14 +126,17 @@ object SparkRInterpreter {
   }
 }
 
-class SparkRInterpreter(process: Process, backendInstance: Any, backendThread: Thread,
-  sparkMajorVersion: String) extends ProcessInterpreter(process) {
+class SparkRInterpreter(process: Process,
+    backendInstance: Any,
+    backendThread: Thread,
+    val sparkMajorVersion: String)
```

Comment:
this is exposed to be used by `session` to differentiate SparkR `setJobGroup` API, do we have any other way to get spark major version?